packages/headless/src/utils/query-expression/near/near.ts

```diff
@@ -2,12 +2,26 @@ import {getNegationPrefix, Negatable} from '../common/negatable';
 import {Part} from '../common/part';
 
 export interface NearExpression extends Negatable {
+  /**
+   * The reference term.
+   */
   startTerm: string;
+
+  /**
+   * The other terms to check against the reference term.
+   */
   otherTerms: OtherTerm[];
```

Comment:
I think we should give a "story" sort of example in those.

I'm not familiar with how those epxressions work, but I would think something like:
> If somebody wanted to search for a tempered glass screen protector for their phone but wanted to match both "iPhone screen protector (tempered glass)" and "tempered glass screen protector for iPhone", they could use:
> `{ startTerm: "screen protector", ["iPhone", "tempered glass"] }`

Don't use my exact example without checking with doc and search API though.

Reply:
Ok, I think I will just to link to the [example in the documentation](https://docs.coveo.com/en/1814/searching-with-coveo/search-prefixes-and-operators#near) to help someone to learn more. Thanks,